CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 You can find and compare releases at the [GitHub release page](https://github.com/nuwave/lighthouse/releases).
 
 ## Unreleased
-
+- Ensure the `@with` directive works properly with polymorphic relations https://github.com/nuwave/lighthouse/pull/1517
```

Comment:
Can we still target this fix towards 4.x, too?

Reply:
Let me know if I placed it in the right place now.

Reply:
Looks better now. As this is a bug fix, do you think we can target 4.x with it?

Reply:
Yes I think we should.